Sources/TokamakCore/Tokens/Color.swift

```diff
@@ -24,26 +24,26 @@ public struct Color: Equatable {
   public let red: Double
   public let green: Double
   public let blue: Double
-  public let alpha: Double
+  public let opacity: Double
```

Comment:
Woah, this uncovers a lot of incompatibilities with SwiftUI, looks like this type is again vestigial from the React implementation... No need to modify it in this PR though, just a note to self 🙂

Reply:
I had to modify it in this PR because there’s a call to `Color` in the demo code which wouldn’t compile for SwiftUI without this change.

Reply:
I understand, thanks for cleaning that up.